lib/reply.js

```diff
@@ -10,6 +10,8 @@ const flatstr = require('flatstr')
 const fastseries = require('fastseries')
 const runHooks = fastseries()
 
+const isError = Symbol('is-error')
+
 function Reply (req, res, context, request) {
   this.res = res
   this.context = context
```

Comment:
can you set this[isError] = false here?

Reply:
Okay, I can do that. Was afraid that extra statement would add some performance cost.

I guess it also needs to be added to `buildReply`?

Reply:
Yes of course.

Reply:
Done